src/bitbucket/BitbucketAPI.ts

```diff
@@ -11,7 +11,7 @@ const BBAPIBaseUrl = 'https://api.bitbucket.org/2.0/repositories';
 
 type MergePullRequestResult = {
   status: string;
-  reason?: string;
+  reason?: any;
```

Comment:
can we extend `reason` to be a union of 2 types if it's no longer just string

Reply:
which 2 types? `reason` will be the object response returned

Reply:
Think Brian is asking for something like 
```
reason?:  { error: { 
message?: string;
fields?: {
  merge_checks: string[]
 } 
}}
```